Model/UserManager.php

```diff
@@ -22,8 +22,8 @@
  */
 abstract class UserManager implements UserManagerInterface
 {
-    private $passwordUpdater;
-    private $canonicalFieldsUpdater;
+    protected $passwordUpdater;
+    protected $canonicalFieldsUpdater;
```

Comment:
The downside of changing the visibility to `protected` is that it becomes a nightmare to remove any of these properties in the future. What about adding a getter method instead?